### ISSUE TITLE
Training from scratch on UCF101, the accuracy is lower than expected.

### ISSUE THREAD

**Issue by jiehg** [facebook/C3D#42]
Hi Du,

I follow "C3D User Guide" and try the example of training from scratch on UCF101. I only modify the batch size from 30 to 10 because of the limited GPU memory. The program runs successfully. But after I run "sh test_ucf101.sh", the clip accuracy on testing data is 0.4022 which is mush lower than the 0.45 proposed in the guide. I also test the clip accuracy on training data, which is 0.9538.
Is my result reasonable?
Thanks.

**Comment by dutran** [comment]
As you reduce batch size by 3, you could reduce learning rate by 3, increase stepsize and max_iter by 3.
Let me know if that works for you.

**Comment by jiehg** [comment]
I try it as you say, the result is satisfactory. I draw the change of "Test score #0" based on "iteration". In the end, the test accuracy is close to 0.45.
![2015-11-22 4 33 48](https://cloud.githubusercontent.com/assets/7336806/11322780/d83f38b0-9136-11e5-93fe-b01e23d6b8ec.png)
Thanks for your help.

**Comment by dutran** [comment]
Glad to hear, good luck with your experiments.

**Comment by holyseven** [comment]
Hi @dutran and @jiehg, I am so confused that if we train UCF101 from scratch, we could only get 45% accuracy on test data but nearly 100% accuracy on training data, isn't it overfitting?

**Comment by dutran** [comment]
Yes, it is known to be overfiting a lot on small datasets like UCF101
